src/easylink/steps/rl-dummy/removing_records/dummy_removing_records.py

```diff
@@ -0,0 +1,44 @@
+# STEP_NAME: removing_records
+
+# REQUIREMENTS: pandas==2.1.2 pyarrow pyyaml
+
+# PIPELINE_SCHEMA: main
+
+import logging
+import os
+
+import pandas as pd
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s %(message)s",
+    handlers=[logging.StreamHandler()],
+)
+diagnostics = {}
+
+
+def load_file(file_path, file_format=None):
+    if file_format is None:
+        file_format = file_path.split(".")[-1]
+    if file_format == "parquet":
+        return pd.read_parquet(file_path)
+    raise ValueError()
+
+
+# LOAD INPUTS and SAVE OUTPUTS
+
+datasets_var = os.environ["INPUT_DATASETS_FILE_PATHS"]
+# don't need to load ids_to_remove since its empty for dummy impl
```

Comment:
I feel like we shouldn't assume this here, better to actually load it and actually drop the IDs in it (if any)